styles.css

```diff
@@ -103,6 +107,209 @@ nav ul li {
   fill: #7f8cff;
 }
 
+.works {
+  display: grid;
+  place-items: center;
+  margin-left: 2rem;
+  grid-template-columns: 90%;
+  grid-template-rows: 4;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 2px solid #dfe1e6;
+  border-radius: 1rem;
+  background: #fff;
+  margin-bottom: 3rem;
+  padding: 1.2rem;
+}
+
+span img {
+  width: 3rem;
+  height: 3rem;
+  margin-right: 1.5rem;
+}
+
+.card img {
+  width: 100%;
+  height: auto;
+  margin-bottom: 0.8rem;
+}
+
+.card h2 {
+  font-size: 2rem;
+  line-height: 2.3rem;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.card span {
+  display: inline-block;
+  margin-bottom: 0.1rem;
+}
+
+.card p {
+  color: #344563;
+  font-size: 1rem;
+  line-height: 1.2rem;
+  margin-bottom: 2rem;
+}
+
+.tech-stack ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.tech-stack li {
+  color: #6070ff;
+  background: #e5e5e5;
+  border-radius: 0.5rem;
+  margin-right: 1rem;
+  padding: 0.2rem;
+  margin-bottom: 1.2rem;
+}
+
+.no-bullet {
+  font-size: 1.3rem;
+  font-weight: 600;
+  color: #344563;
+  line-height: 1.5rem;
+  margin-right: 0.5rem;
+}
+
+.project-tag {
+  color: #7a869a;
+  font-size: 0.8rem;
+  line-height: 1.5rem;
+  margin-right: 0.3rem;
+  list-style-type: disc;
+  list-style-position: inside;
+}
+
+.btn {
+  font-weight: 500;
+  font-size: 1.2rem;
+  line-height: 1.9rem;
+  border: 1px solid #396df2;
+  border-radius: 0.5rem;
+  background: white;
+  text-align: center;
+  padding: auto;
+  margin-right: auto;
+  height: 3rem;
+  color: #396df2;
+  width: 8rem;
+  display: flex;
+  align-items: center;
+  align-content: center;
+}
+
+.btn:hover {
+  color: white;
+  background: #6070ff;
+}
+
+.btn:active {
+  color: white;
+  background: #2230d2;
+}
+
+.btn:disabled {
+  color: #5e6c84;
+  background: #e5e5e5;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  justify-content: center;
+  margin: auto;
+  background-color: white;
+}
+
+.resume-btn {
+  width: auto;
+}
+
+.languages-title {
+  font-weight: 500;
+  font-size: 1.8rem;
+  line-height: 2rem;
+  color: #000;
+}
+
+.about-list {
+  display: flex;
+  flex-direction: column;
+  margin-top: 1.3rem;
+}
+
+.langs li {
+  width: 90%;
+  height: 4rem;
+  background: #f7f7f9;
+  margin-bottom: 0.5rem;
+  display: flex;
+  align-content: baseline;
+  border-radius: 1.2rem;
```

Comment:
- [ ] I would kindly suggest you to reduce the width  in line 255 and font size, what if you can reduce the width to 50% and see how its .according to project requirement your design you look exact like figm design template you choose    
- [ ] figma design
![just like this](https://user-images.githubusercontent.com/92458236/235265026-02b95252-c594-413f-a2ca-d762e7131955.jpg)

- [ ] your design
![you design](https://user-images.githubusercontent.com/92458236/235265082-a815d5a7-5723-42cd-b0e9-89ed856baa1a.jpg)